src/pyhf/cli/stats.py

```diff
@@ -0,0 +1,67 @@
+import logging
+
+import click
+import json
+
+from ..utils import hypotest, EqDelimStringParamType
+from ..workspace import Workspace
+from .. import tensorlib, set_backend, optimize
+
+logging.basicConfig()
+log = logging.getLogger(__name__)
+
+
+@click.group(name='stats')
+def cli():
+    pass
+
+
+@cli.command()
+@click.argument('workspace', default='-')
+@click.option(
+    '--output-file',
+    help='The location of the output json file. If not specified, prints to screen.',
+    default=None,
+)
+@click.option('--measurement', default=None)
+@click.option('-p', '--patch', multiple=True)
+@click.option('--testpoi', default=1.0)
+@click.option('--teststat', type=click.Choice(['q', 'qtilde']), default='qtilde')
+@click.option('--optimizer')
+@click.option('--optconf', type=EqDelimStringParamType(), multiple=True)
+def cls(
+    workspace, output_file, measurement, patch, testpoi, teststat, optimizer, optconf
+):
+    with click.open_file(workspace, 'r') as specstream:
+        wspec = json.load(specstream)
+
+    w = Workspace(wspec)
```

Comment:
Use consistent variables names with `inspect`
```suggestion
    ws = Workspace(wspec)
```